rand_core/Cargo.toml

```diff
@@ -27,5 +27,8 @@ serde1 = ["serde"] # enables serde for BlockRng wrapper
 serde = { version = "1", features = ["derive"], optional = true }
 getrandom = { version = "0.1", optional = true }
 
+[build-dependencies]
+version_check = "0.9"
```

Comment:
Personally I would like avoid this build dependency and use [`rustdoc-args = ["--cfg", "docsrs"]`](https://github.com/RustCrypto/traits/blob/master/digest/Cargo.toml#L23-L25) (maybe with a different name) in Cargo.toml instead. This approach will work fine for docs.rs generation, while for manual generation it is still possible to pass this option.

Reply:
[version_check](https://crates.io/crates/version_check) is a small crate without sub-dependencies and only used at build-time — but sure, we can use that instead.

Reply:
We'd better add a feature flag for this. It breaks `--all-features` on stable compilers, but that's not terrible.

Reply:
No, it's compatible. As I've mentioned in the previous message, you just have to pass the `cfg` option directly:
```
RUSTDOCFLAGS="--cfg docsrs" cargo doc --open --all-features
```
I guess we could use `doc_cfg` or something similar instead of `docsrs` for the config name.

I think the cfg option is similarly "not terrible", so since it does not influence code generation, only docs, using features here is a bit inappropriate for my taste. But I don't have a strong opinion on this matter, so I am fine with both options.